### PR TITLE
new attributes & fix's

### PR DESCRIPTION
**Pull request by AkioSarkiz**
add SecureSchema and Server attributes
add target for Controller attribute
add auto-fix route path

```php
#[
    Info('app', '1.0.0'),
    Server('test', 'url'),
    Server('test', 'url2'),
    SchemaSecurity(
        'bearerAuth',
        'http',
        'bearerAuth',
        'header',
        'JWT',
        'bearer',
    ),
]
class Controller
{
    //
}

### CODE REVIEW

File: src/Generator.php [hunk 1]
```diff
@@ -4,16 +4,49 @@
...
+    }
+
+    /**
+     * @return Generator
+     */
+    #[Pure]
+    public static function factory(): Generator
+    {
+        return new self(new GeneratorHttp, new GeneratorSchemas);
+    }
```
Comment (uderline) [inline src/Generator.php:49]: The constructor should be public to test it - avoiding reflections which are a pain - and the factory method should be an actual factory class. Soon, I'd like it to generate the scheme in json and yaml so the factory will be ready

File: src/Generator.php [hunk 2]
```diff
@@ -25,50 +58,115 @@ class Generator
...
-
         $classes = get_declared_classes();
+
         foreach ($classes as $class) {
             try {
                 $reflectionClass = new ReflectionClass($class);
             } catch (ReflectionException $e) {
-                continue;
-            }
+                echo '[Waring] ReflectionException ' . $e->getMessage();
```
Comment (uderline) [inline src/Generator.php:67]: Typo in "Warning"

**Review by uderline — CHANGES_REQUESTED** [review]
Thanks for your help

File: src/Generator.php (hunk 1 above)
Comment (AkioSarkiz) [reply src/Generator.php:49]: ok, i rename this method to `create`

**Comment by AkioSarkiz** [comment]
fixed conflicts

File: src/Generator.php (hunk 1 above)
Comment (uderline) [reply src/Generator.php]: Last comment, could you just add parenthesis at the end of `GeneratorHttp` and  `GeneratorSchemas` please ? (PSR12)

Comment (AkioSarkiz) [reply src/Generator.php]: done

**Comment by AkioSarkiz** [comment]
If possible, then release some version. Thanks 🙂

**Comment by uderline** [comment]
Done !

**Pull request merged**